ouroboros-consensus-cardano/src/unstable-shelley-testlib/Test/Consensus/Shelley/Examples.hs

```diff
@@ -169,7 +182,7 @@ fromShelleyLedgerExamples
         ]
     results =
       labelled
-        [ ("LedgerTip", SomeResult GetLedgerTip (blockPoint blk))
+        [ ("LedgerTip", SomeResult GetLedgerTip (blockPoint blkWithoutCert))
```

Comment:
We probably want to make `results` parametric on `blk` to avoid hardcoding the `blockPoint`